homeassistant/helpers/config_validation.py

```diff
@@ -653,6 +654,13 @@ def validator(value):
     return validator
 
 
+WHITELIST = [
+    re.compile(CONF_NAME),
```

Comment:
Let's not compile these regexes at the root level, let's move them inside the function. Also the name `WHITELIST` inside a 1000 line file is ambiguous, so let's just get rid of it and inline it.

Reply:
Fixed.